### PR TITLE
Correct indentation for a couple libraries

### PR DESCRIPTION
**Pull request by erasche**
They were populating the top level directories, rather than existing within their topic

### CODE REVIEW

**Comment by erasche** [comment]
I guess I will leave this open until we fix the [other ones too](https://build.galaxyproject.eu/job/usegalaxy-eu/job/shared-data/3/console) :(

**Comment by erasche** [comment]
@bgruening fixed the rest, if you have time to check.

File: topics/transcriptomics/tutorials/network-analysis-with-heinz/data-library.yaml [hunk 1]
```diff
@@ -5,153 +5,158 @@ destination:
...
-    url: https://zenodo.org/record/1344105/files/3307_CN_DZ_PairTo_3306.txt
+  - name: Network analysis with Heinz
+    items:
+    - name: 'DOI: 10.5281/zenodo.1344105 CP'
+      description: Caries Postive (Disease)
+      items:
+      - ext: txt
+        info: https://doi.org/10.5281/zenodo.1344105
+        src: url
+        url: https://zenodo.org/record/1344105/files/2241_CP_DZ_PairTo_2242.txt
```
Comment (bgruening) [inline topics/transcriptomics/tutorials/network-analysis-with-heinz/data-library.yaml]: I guess this can be tabular - all.

**Comment by bgruening** [comment]
Gracias!

**Pull request merged**